trulens_eval/examples/frameworks/llama_index/llama_index_complex_evals.ipynb

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    sentence_index = VectorStoreIndex.from_documents(
</pre><p>We probably should download + load this index from storage, similar to the sentence index from notebook 2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>